packages/collections/_test.pony

```diff
@@ -663,7 +661,18 @@ class \nodoc\ iso _TestRange is UnitTest
     let range: Range[N] = Range[N](min, max, step)
     let range_str = "Range(" + min.string() + ", " + max.string() + ", " + step.string() + ")"
 
-    h.assert_true(range.is_infinite(), range_str + " should be infinite")
+    h.assert_true(range.is_infinite(), range_str + " should be infinite")    
+    try
```

Comment:
Since the PR makes that if `range.is_infinite() == true`, then `range.has_next() == true`, perhaps the test would be clearer if we tested that, instead of testing that `range.next()?` doesn't raise an error?

Reply:
I can make the change you suggested. However, the reason why I tested not for the new, expected behavior but for the absence of the old, erroneous one is that I was asked to create a test that prevents regressions. Should I nonetheless just test for the simple `range.has_next() == true`?

Reply:
Ah, you're right about the regression issue, let's leave the test as it is then.